sunspec/core/modbus/mbmap.py

```diff
@@ -22,12 +22,17 @@
 """
 
 import struct
+import sys
 
 try:
     import xml.etree.ElementTree as ET
 except:
     import elementtree.ElementTree as ET
 
+# Python 3 compatibility for long()
+if sys.version_info > (3,):
+    long = int
```

Comment:
Yes but to keep from making the format strings dynamic based on the input variable it is simpler to make the variable match the existing format strings.